### PR TITLE
Shibboleth role and playbook example

### PR DESCRIPTION
**Pull request by fmarco76**

### CODE REVIEW

File: Ansible/host_vars/burns.ct.infn.it [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+idp_authn_LDAP_baseDN: 'ou=People,dc=local'
+
+# LDAP user filter
+idp_authn_LDAP_userFilter: '(cn={user})'
+
+# LDAP bind DN
+idp_authn_LDAP_bindDN: 'cn=fmarco76,ou=People,dc=local'
+
+# LDAP bind DN credentials
+idp_authn_LDAP_bindDNCredential: 'c9SF4eudh'
```
Comment (brucellino) [inline Ansible/host_vars/burns.ct.infn.it]: Remove this sensitive password

File: Ansible/shibboleth-idp-v3.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+- hosts: all
+  name: Configure Shibboleth Identity Provider
+  environment:
+    JAVA_HOME: /usr/lib/jvm/default-java
+  pre_tasks:
+    - name: Install pip
```
Comment (brucellino) [inline Ansible/shibboleth-idp-v3.yml]: "install" should be "ensure"

File: Ansible/shibboleth-idp-v3.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      become: yes
+      apt: name='python-pip' update_cache=yes
+
+    - name: Update pip
```
Comment (brucellino) [inline Ansible/shibboleth-idp-v3.yml]: Why do you need to have two tasks here ? Shouldn't apt install the latest pip ?

Comment (fmarco76) [inline Ansible/shibboleth-idp-v3.yml]: Unfortunately the pip version installed in debian is an old version

File: Ansible/shibboleth-idp-v3.yml (hunk 2 above)
Comment (fmarco76) [inline Ansible/shibboleth-idp-v3.yml]: OK

**Pull request merged**